flink-runtime/src/test/java/org/apache/flink/runtime/io/network/netty/ServerTransportErrorHandlingTest.java

```diff
@@ -101,7 +104,22 @@ public void channelRead(ChannelHandlerContext ctx, Object msg)
         NettyTestUtil.NettyServerAndClient serverAndClient = null;
 
         try {
-            serverAndClient = initServerAndClient(protocol, createConfig());
+            for (int retry = 0; retry < NETTY_INIT_MAX_RETRY_TIMES; retry++) {
+                try {
+                    serverAndClient = initServerAndClient(protocol, createConfig());
+                    break;
+                } catch (Exception e) {
+                    if (retry >= NETTY_INIT_MAX_RETRY_TIMES - 1) {
+                        throw e;
```

Comment:
Maybe we should give more details in the `Exception`. Such as the retry failed. WDYT?

Reply:
Thanks a lot for reviewing. @guoweiM   ok，I will fix it recently.